utils/xmlservice-cli.c

```diff
@@ -158,10 +180,13 @@ int main()
 
     rc = _ILECALL(&xmlstoredp, &arglist.base, signature, RESULT_UINT8);
 
+    /* perror likely not useful w/ _ILECALL */
     if (rc != ILECALL_NOERROR) {
+        dprintf(olderr, "%s: _ILECALL error\n", argv[0]);
```

Comment:
Didn't know this function existed, very cool. Though AIX only has `drpintf` :joy:

![image](https://user-images.githubusercontent.com/7004340/185973386-819db2c7-3dc8-4adc-85c1-1d9ed269f805.png)